### PR TITLE
Adventures in LookupServer, pt. 4

### PR DESCRIPTION
**Pull request by bugaevc**
Well, hello friends! :^)

This is the part 4, where we get basic mDNS support :tada:

screen

Caveat usor: the mDNS implemention is not complete or comprehensive; it's just enough to get `.local` name resolution working. It should be a bit less terrible than the [Terrible mDNS responder](https://github.com/bugaevc/terrible-mdns-responder) though :smile:

### CODE REVIEW

File: Userland/Services/LookupServer/MulticastDNS.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    Vector<IPv4Address> local_addresses() const;
+
+    DNSName m_hostname;
+
+    static constexpr sockaddr_in mdns_addr {
+        AF_INET,
+        // htons(5353),
+        0xe914,
+        { // 224.0.0.251
+            0xfb0000e0 },
```
Comment (bugaevc) [inline Userland/Services/LookupServer/MulticastDNS.h:41]: clang-format made me do this

Comment (IdanHo) [reply Userland/Services/LookupServer/MulticastDNS.h:41]: putting a trailing comma after the last entry should make it format correctly :)

Comment (alimpfard) [reply Userland/Services/LookupServer/MulticastDNS.h:41]: Would this work?
```suggestion
// 224.0.0.251
{ 0xfb0000e0 },
```

Comment (bugaevc) [reply Userland/Services/LookupServer/MulticastDNS.h:41]: Seems to work, thx!

**Pull request merged**